gem/chhaajja/videos/models.py

```diff
@@ -0,0 +1,46 @@
+from django.db import models
+
+from wagtail.core.models import Page
+from wagtail.core.fields import RichTextField
+from wagtail.admin.edit_handlers import FieldPanel
+from wagtail.search import index
+
+class VideosIndexPage(Page):
+    intro = RichTextField(blank=True)
+
+    content_panels = Page.content_panels + [
+        FieldPanel('intro', classname="full")
+    ]
+
+    def get_context(self, request):
+        context = super().get_context(request)
+
+        # Add extra variables and return the updated context
+        context['videos_entries'] = VideosPage.objects.live()
+        context['latest_video'] = VideosPage.objects.latest('id')
+        return context
+
+
+
+
+# Keep the definition of BlogIndexPage, and add:
+
+
+class VideosPage(Page):
```

Comment:
It would be better if this model inherited from a model like `ArticlePage' from https://github.com/praekeltfoundation/molo/blob/develop/molo/core/models.py#L1574

In order for admins to not see all the unused fields, the content panels can be overridden.

This is if commenting or translatable pages are necessary on the pages, and so that the pages are uniform across sites.